File: clients/java3/src/org/openamq/client/ConnectionTuneParameters.java
package org.openamq.client;

/**
 * @author Robert Greig (robert.j.greig@jpmorgan.com)
 */
public class ConnectionTuneParameters
{
    private long _frameMax;

    private long _channelMax;

    private long _handleMax;

    private long _hearbeat;

    private long _txnLimit;

    public long getFrameMax()
    {
        return _frameMax;
    }

    public void setFrameMax(long frameMax)
    {
        _frameMax = frameMax;
    }

    public long getChannelMax()
    {
        return _channelMax;
    }

    public void setChannelMax(long channelMax)
    {
        _channelMax = channelMax;
    }

    public long getHandleMax()
    {
        return _handleMax;
    }

    public void setHandleMax(long handleMax)
    {
        _handleMax = handleMax;
    }

    public long getHearbeat()
    {
        return _hearbeat;
    }

    public void setHearbeat(long hearbeat)
    {
        _hearbeat = hearbeat;
    }

    public long getTxnLimit()
    {
        return _txnLimit;
    }

    public void setTxnLimit(long txnLimit)
    {
        _txnLimit = txnLimit;
    }
}

File: dotnet/xmsclient/Client/ConnectionTuneParameters.cs
using System;
using jpmorgan.mina.common;
using log4net;

namespace OpenAMQ.XMS.Client
{
    public class ConnectionTuneParameters
    {
        private uint _frameMax;

        private ushort _channelMax;

        private uint _hearbeat;

        private uint _txnLimit;

        public uint FrameMax
        {
            get
            {
                return _frameMax;
            }
            set
            {
                _frameMax = value;
            }
        }

        public ushort ChannelMax
        {
            get
            {
                return _channelMax;
            }
            set
            {
                _channelMax = value;
            }
        }

        public uint Heartbeat
        {
            get
            {
                return _hearbeat;
            }
            set
            {
                _hearbeat = value;
            }
        }

        public uint TxnLimit
        {
            get
            {
                return _txnLimit;
            }
            set
            {
                _txnLimit = value;
            }
        }
    }
}
